scripts/capture.py

```diff
@@ -0,0 +1,31 @@
+import cv2
+import datetime
+import os
+
+OUTPUT_DIR = '/home/pi/data'
+DIR = '%Y-%m-%d'
+FILE = '%H:%M:%S.png'
+
+cap = cv2.VideoCapture(0)
+
+#  warmup camera
+for i in range(15):
+    _, _ = cap.read()
+
+ret, frame = cap.read()
+grayimg = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+
+_, thresh = cv2.threshold(grayimg, 80, 255, cv2.THRESH_BINARY)
+_, thresh_inv = cv2.threshold(grayimg, 80, 255, cv2.THRESH_BINARY_INV)
+nonzero = cv2.countNonZero(thresh)
+nonzero_inv = cv2.countNonZero(thresh_inv)
+
+if nonzero - nonzero_inv > 0:
```

Comment:
Well I think we definitely shouldn't do two thresholds. We are effectively asking here how many pixels there are that have lower intensity value that 80 from all the pixels. So same effect can be probably achieved by just comparing it to 1/3 of total number of pixels or by replacing the second threshold with numpy operation something like `np.count_nonzero(grayimg < 80)`. Or you can just simply ask what is the highest intensity pixel with max and if it is less then lets say 20 then everything else is lower therefore even blacker.